### ISSUE TITLE
Total time for filtered lists

### ISSUE THREAD

**Issue by AlexSoldevila** [chri55w/QuickBooks-Timer#91]
Hi Chris,
Just something Stephen and I thought about while reviewing projects. It could be a "nice to have" feature that, in the status bar, if a filter has been applied to the list of activities, it would show the Total for the filtered list.

F.ex. we were trying to figure out how much time we spent on VDH support this month, so I filtered the list and it worked fantastic. However, it would be the icing of the cake if I could easily see the sum of those times.

What do you think?

<img width="531" alt="totalfiltered" src="https://cloud.githubusercontent.com/assets/19147557/15424913/736f299a-1e85-11e6-9ce6-5328c09f19d2.PNG">

**Comment by chri55w** [comment]
Yes, this is actually something that Steve has also pointed out, the idea of the Total Time is to show the total time of the Time Logs on screen however its binding to the Collection of Time Logs does not take into consideration the filters that are applied so it doesn't update the Total Time. Its not an entirely straightforward change but its an issue that id like to address in the next version!
